### PR TITLE
- small tweaks to component layout alignment

### PR DESCRIPTION
**Pull request by jmurray-d2l**

### CODE REVIEW

**Comment by jmurray-d2l** [comment]
Still waiting on design review, but opening this up to code review now since it may not be until later today until I can get some one from design to take a look

File: components/left-panel/consistent-evaluation-submissions-page.js [hunk 1]
```diff
@@ -36,13 +36,13 @@ export class ConsistentEvaluationSubmissionsPage extends SkeletonMixin(RtlMixin(
 			:host([hidden]) {
 				display: none;
 			}
-			d2l-consistent-evaluation-submission-item {
-				margin: 0.5rem;
-				padding: 0.5rem;
+			.d2l-consistent-evaluation-submission-list-view-skeleton {
```
Comment (jmurray-d2l) [inline components/left-panel/consistent-evaluation-submissions-page.js:39]: this just makes sure the skeleton is not drawn after loading is finished

**Comment by jmurray-d2l** [comment]
The doc I have been following if you are interested https://d2lmail-my.sharepoint.com/:w:/g/personal/swu_desire2learn_com/EdaRF9R-2AlBqy4iomqJtqQBkKNyVBxIuu3HbrFkmV_qrg?e=T51egn

File: components/header/d2l-consistent-evaluation-learner-context-bar.js [hunk 2]
```diff
@@ -83,20 +83,20 @@ export class ConsistentEvaluationLearnerContextBar extends SkeletonMixin(RtlMixi
 			}
 			@media (max-width: 929px) and (min-width: 768px) {
```
Comment (jeroach) [inline components/header/d2l-consistent-evaluation-learner-context-bar.js:84]: I think you're missing updating the `padding-left` of LCB to `37px` on the default size (previously it was `30px`)

File: components/right-panel/consistent-evaluation-right-panel.js [hunk 3]
```diff
@@ -85,6 +85,14 @@ export class ConsistentEvaluationRightPanel extends LocalizeConsistentEvaluation
 		};
 	}
 
+	static get styles() {
+		return  css`
+			:host {
+				margin: 1.5rem 1.2rem 0 1.2rem;
```
Comment (jeroach) [inline components/right-panel/consistent-evaluation-right-panel.js:91]: For right panel, 616px - 767px - I think we want left/right margin = `18px` or `0.9rem`

**Review by jeroach — APPROVED** [review]

**Review by AlexBedley — APPROVED** [review]

**Pull request merged**